src/js/shared/Events/index.mjs

```diff
@@ -229,7 +229,6 @@ export const prioritize = function(...args) {
 }
 
 const unsubscribe = (key) => {
-  console.log(key)
```

Comment:
Do you really want this `console.log` here?

Reply:
no, that's why I took it out.

Reply:
Oh my bad...jeez, you took it out. I read it as an addition.